packages/cozy-sharing/src/components/ShareDialogCozyToCozy.jsx

```diff
@@ -88,16 +89,18 @@ const ShareDialogCozyToCozy = ({
         </div>
       }
       actions={
-        <DumbShareByLink
-          checked={link !== null}
-          document={document}
-          documentType={documentType}
-          link={link}
-          onChangePermissions={onUpdateShareLinkPermissions}
-          onDisable={onRevokeLink}
-          onEnable={onShareByLink}
-          permissions={permissions}
-        />
+        showShareByLink ? (
+          <DumbShareByLink
+            checked={link !== null}
+            document={document}
+            documentType={documentType}
+            link={link}
+            onChangePermissions={onUpdateShareLinkPermissions}
+            onDisable={onRevokeLink}
+            onEnable={onShareByLink}
+            permissions={permissions}
+          />
+        ) : null
```

Comment:
This part is not enough as cozy-ui still display the `actions` container even if empty. I may have to edit cozy-ui later.

Reply:
What about not setting `actions` to null? Does UI still display the container if the props is not specified?

Reply:
Fixed with : https://github.com/cozy/cozy-libs/pull/1326#issuecomment-893482019